pandas/tests/computation/test_eval.py

```diff
@@ -1283,10 +1282,8 @@ def test_assignment_column(self):
         msg = "left hand side of an assignment must be a single name"
         with pytest.raises(SyntaxError, match=msg):
             df.eval("d,c = a + b")
-        if compat.PY38:
```

Comment:
could maybe leave these changes for a follow-on cleanup to keep the diff here smaller? Reviewers could then easily identify where/if tests have been changed.